contrib/test_jenkins.sh

```diff
@@ -371,6 +371,27 @@ build_icc() {
 	module unload intel/ics
 }
 
+#
+# Build with PGI compiler
+#
+build_pgi() {
+	echo 1..1 > build_pgi.tap
+	if module_load pgi/19.7 && pgcc18 --version
+	then
+		echo "==== Build with PGI compiler ===="
+		# PGI failed to build valgrind headers, disable it for now
+		../contrib/configure-devel --prefix=$ucx_inst CC=pgcc18 --without-valgrind
+		$MAKEP clean
+		$MAKEP
+		$MAKEP distclean
+		echo "ok 1 - build successful " >> build_pgi.tap
+	else
+		echo "==== Not building with PGI compiler ===="
+		echo "ok 1 - # SKIP because PGI compiler not installed" >> build_pgi.tap
+	fi
+	module unload intel/ics
```

Comment:
unload pgi

Reply:
copy-paste issue :)
fixed

Reply:
also, think should use pgi-latest module (which is symlink to 19 today) and not specific version, so we can upgrade pgi and keep testing with latest

Reply:
technically yes, but in this case we have risk to break testing on update environment.
setting specific compiler version allows us to migrate to newer compiler after all fixes for new compiler are merged